versions.json

```diff
@@ -136,14 +136,15 @@
         "fpm_directories": ["include","lib"]
     },
     "cpr": {
-        "commitish": "1.3.0",
-        "version_string": "1.3.0",
+        "git_repository": "https://github.com/libcpr/cpr",
```

Comment:
Once you're happy with it, we'll get @trel or @alanking to update our fork. Then we can remove the ref to their git repo.

Reply:
Since we're not patching, I don't see a reason we shouldn't use the upstream repo

Reply:
That's correct. We could use the upstream repo, but I believe we maintain forks for all externals to protect us from issues with upstream.

@trel would know the exact reason for why we do that.

Always open to hearing how to make this better.